extensions/amp-analytics/0.1/variables.js

```diff
@@ -212,17 +156,11 @@ export class VariableService {
 
     const replacementPromises = [];
     let replacement = template.replace(/\${([^}]*)}/g, (match, key) => {
-
-      // Note: The parsing for variables breaks when `|` is used as
-      // something other than the filter separator. A full-fledged parser would
-      // be needed to fix this.
-      const tokens = key.split('|');
-      const initialValue = tokens.shift().trim();
-      if (!initialValue) {
+      if (!key) {
         return Promise.resolve('');
       }
 
-      const {name, argList} = this.getNameArgs_(initialValue);
+      const {name, argList} = this.getNameArgs_(key);
```

Comment:
I believe args are no longer needed since we remove old filter syntax.

Reply:
I think we need to keep this in case people are doing stuff like [this](https://github.com/ampproject/amphtml/blob/6f3a87746c59fec66b9adb80b7fdc6ec972d807c/extensions/amp-analytics/0.1/test/test-variables.js#L109).